examples/python/aoa.py

```diff
@@ -0,0 +1,98 @@
+# Originally developed by JSBSim Team
+# Modified by Guilherme A. L. da Silva - aerothermalsolutions.co
+# Calculation required by aircraft icing enginering
+
+import jsbsim
+from IPython.core.display import display, HTML
+import xml.etree.ElementTree as ET
+import matplotlib.pyplot as plt
+import matplotlib.style as style
+style.use('fivethirtyeight')
```

Comment:
```suggestion
```
Such customizations should not be used in an example to keep things as simple as possible.